Tests/RxSwift/filterMapTests.swift

```diff
@@ -12,14 +12,20 @@ import RxSwift
 import RxSwiftExt
 import RxTest
 
-enum FilterMapTestError: Error {
-    case error
-}
+final class FilterMapTests: XCTestCase {
+    private var scheduler: TestScheduler!
 
-class FilterMapTests: XCTestCase {
-    func testIgnoreEvenAndEvenizeOdds() {
+    override func setUp() {
+        super.setUp()
+        scheduler = TestScheduler(initialClock: 0)
+    }
+
+    override func tearDown() {
+        scheduler = nil
```

Comment:
not a huge deal but you don't really need this (a new one overrides the old one)

Reply:
It's needed for resource deallocation after last test from this class
https://qualitycoding.org/xctestcase-teardown/